GameOfLifeConsole/Services/FileReadSave.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+
+namespace GameOfLifeConsole.Services
+{
+    /// <summary>
+    /// To read and save the data on hard disc.
+    /// </summary>
+    class FileReadSave
+    {
+        // The path to which the data will be saved and restored. The variable.
+        private string Path = $"{Environment.CurrentDirectory}\\MyFile.json";
+        private int[,] gameField;
```

Comment:
private properties should start with underscore and should continue with lower camelCase.

Reply:
Got it. Thank you. Done.